antora.yml

```diff
@@ -106,3 +106,6 @@ asciidoc:
     theia-endpoint-image: eclipse/che-theia-endpoint-runtime:next
     url-devfile-registry-repo: https://github.com/eclipse/che-devfile-registry
     url-plug-in-registry-repo: https://github.com/eclipse/che-plugin-registry
+    all-namespaces-olm-channel: tech-preview-stable-all-namespaces
+    all-namespaces-olm-package: eclipse-che
+    stable-olm-catalog-source: community-operators
```

Comment:
Pls provide downstream specific values in the description

Reply:
Done.